src/main/java/uk/gov/companieshouse/itemhandler/kafka/EmailSendKafkaProducer.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.companieshouse.itemhandler.kafka;
+
+import org.springframework.beans.factory.InitializingBean;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.kafka.exceptions.ProducerConfigException;
+import uk.gov.companieshouse.kafka.message.Message;
+import uk.gov.companieshouse.kafka.producer.Acks;
+import uk.gov.companieshouse.kafka.producer.CHKafkaProducer;
+import uk.gov.companieshouse.kafka.producer.ProducerConfig;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
+
+import java.util.concurrent.ExecutionException;
+
+import static uk.gov.companieshouse.itemhandler.ItemHandlerApplication.APPLICATION_NAMESPACE;
+
+
+@Service
+public class EmailSendKafkaProducer implements InitializingBean {
+    private static final Logger LOGGER = LoggerFactory.getLogger(APPLICATION_NAMESPACE);
+    private CHKafkaProducer chKafkaProducer;
+    @Value("${spring.kafka.consumer.bootstrap-servers}")
+    private String brokerAddresses;
+
+    /**
+     * Sends message to Kafka topic
+     * @param message message
+     * @throws ExecutionException should something unexpected happen
+     * @throws InterruptedException should something unexpected happen
+     */
+    public void sendMessage(final Message message) throws ExecutionException, InterruptedException {
+        LOGGER.trace("Sending message to kafka");
```

Comment:
Should more details be logged out?

Reply:
I was planning to leave that until GCI-1162. But I suppose we could at least log the topic here?

Reply:
Nah don't worry if it's part of another story, we can do it then